README.md

```diff
@@ -1,24 +1,144 @@
-# README
+# Movie Biskop
```

Comment:
## There is no pull request open on your frontend project, so I'll comment on it here:

- [ ] When I login with my username, it is case sensitive.  Please, consider passing a `toLowerCase` method to it in the frontend to prevent confusion for your users.   eg. "Max" vs "max".
- [ ] Pleae, kindly include instructions in your readme file about how to run the tests.
- [ ] On your frontend readme, your link to the license is broken because it poitns to a file in your project that does not exist  Kindly go [here](https://github.com/microverseinc/readme-template) and grab the `MIT.md` file, include it in the root of your project, and link to it in the "License" section.
- [ ] For your next review request, please open a pull request on your frontend project, so it can be properly reviewed.  The PR must point from dev => main.

That way, your frontend  project will be well documented, and your login UX vastly improved 💪